cockpit_api/pom.xml

```diff
@@ -41,6 +41,10 @@
 					<groupId>org.junit.vintage</groupId>
 					<artifactId>junit-vintage-engine</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>org.skyscreamer</groupId>
```

Comment:
do we need this dependency ?

Reply:
no more need; Kojak added it for his unit tests, removed